Formula/cedille.rb

```diff
@@ -0,0 +1,117 @@
+require "language/haskell"
+
+class Cedille < Formula
+  include Language::Haskell::Cabal
+
+  desc "Language based on the Calculus of Dependent Lambda Eliminations"
+  homepage "https://cedille.github.io/"
+
+  stable do
+    url "https://github.com/cedille/cedille/archive/v1.1.2.tar.gz"
+    sha256 "cf6578256105c7042b99a70a897c1ed60f856b28628b79205eb95730863b71cb"
+
+    resource "ial" do
+      url "https://github.com/cedille/ial/archive/v1.5.0.tar.gz"
+      sha256 "f003a785aba6743f4d76dcaafe3bc08bf879b2e1a7a198a4f192ced12b558f46"
+    end
+  end
+
+  head do
+    url "https://github.com/cedille/cedille.git"
+
+    resource "ial" do
+      url "https://github.com/cedille/ial.git"
+    end
+  end
+
+  depends_on "agda" => :build
+  depends_on "cabal-install" => :build
+  depends_on "ghc" => :build
```

Comment:
`ghc` is probably not a build dependency, but runtime as well:

```
==> /usr/local/Cellar/cedille/1.1.2/bin/cedille-core /private/tmp/cedille-test-20191227-87898-1wxx4mf/core-test.cdle
dyld: Library not loaded: /usr/local/opt/ghc/lib/ghc-8.8.1/directory-1.3.3.2/libHSdirectory-1.3.3.2-ghc8.8.1.dylib
  Referenced from: /usr/local/Cellar/cedille/1.1.2/bin/cedille-core
  Reason: image not found
```

Reply:
Yeah, I had it as a runtime dependency before, I was just curious if it was required with the switch to a statically linked binary.